assets/csidriveroperators/shared-resource/13_config_configmap.yaml

```diff
@@ -0,0 +1,5 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: csi-driver-shared-resource-config
+  namespace: openshift-cluster-csi-drivers
```

Comment:
How is an empty config map useful?

Reply:
It gets updated with a list of namespaces to ignore, we may end up moving this into the shared resources controller in the future.

Reply:
Conversely, I might "flip the script" create watches on demand based on shares that come in, and quit maintaining the ignore list

If I succeed with that, then this will contain keys for the relist interval and refresh resources, and look similar to https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/manifests/03_configmap.yaml or https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/manifests/05_builder-deployer-config.yaml